boards/common/microbit/Kconfig

```diff
@@ -0,0 +1,11 @@
+# Copyright (c) 2021 Inria
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+
+config MODULE_MICROBIT
+    bool "BBC micro:bit specific LED matrix handling"
+    select XTIMER
+    select MINEPLEX
```

Comment:
```suggestion
    select MODULE_XTIMER
    select MODULE_MINEPLEX
```

Reply:
According to Makefile.dep, it should rather be ztimer/ztimer_usec